packages/nexrender-core/src/tasks/actions.js

```diff
@@ -21,10 +21,14 @@ module.exports = actionType => (job, settings) => {
     settings.logger.log(`[${job.uid}] applying ${actionType} actions...`);
 
     return PromiseSerial((job.actions[actionType] || []).map(action => () => {
-        return requireg(action.module)(job, settings, action, actionType).catch(err => {
-            return Promise.reject(new Error(`Error loading ${actionType} module ${action.module}: ${err}`))
-        });
+        if(settings.actions[action.module]){
```

Comment:
@limzykenneth @inlife  Need to check `settings.actions` is defined before calling `settings.actions[action.module]`.